server/.env

```diff
@@ -1 +1,8 @@
 TEAM_MEMBERS="Aecio, Bonnie, Shums, Kassam, Sebastian, Tojo, Ray"
+MONGO_URL=mongodb+srv://angelfish:07jqaI8yKLDkpsUH@teamangelfish.qdkkk.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
+JWT_SECRET=akjhsfjkghkjhdfjkssjkdfjks
+NODE_ENV=develop
+RAPID_API_KEY="b45bedc57emsh23d534a09704e52p1ab1a8jsn764974156db7"
+RAPID_API_HOST="skyscanner-skyscanner-flight-search-v1.p.rapidapi.com"
+AWS_SECRET_ACCESS_KEY="6L9b5bKbr+yCiG2ZD35SKb/AE/8wodAadHQ+7+mS"
+AWS_ACCESS_KEY_ID="AKIASZQUJFTEWXE3JA4R"
```

Comment:
make sure your `.env` is git ignored.
keep in mind you might also have to delete your git cache.
This post by Ray shows the way to handle this:
https://hatchways.slack.com/archives/C01QJMCP37W/p1615248679027400